core-galleon-pack/src/main/resources/feature_groups/elytron-standalone.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="elytron-standalone" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
for consistency sake, we should either name the feature group `standalone-xxx` or `xxx-standalone` (e.g. in the modified file just above, we have `standalone-security-realms`)

Reply:
good point, standalone-elytron would be a better name.